tests/unit/TranslateTest.js

```diff
@@ -52,3 +54,42 @@ describe('translate', () => {
         expect(translate.translate('en', ['testKeyGroup', 'testFunction'], {testVariable})).toBe(expectedValue);
     });
 });
+
+describe('Translation Keys', () => {
+    function traverseKeyPath(source, path, keyPaths) {
+        const pathArray = keyPaths || [];
+        const keyPath = path ? `${path}.` : '';
+        _.each(_.keys(source), (key) => {
+            if (_.isObject(source[key]) && !_.isFunction(source[key])) {
+                traverseKeyPath(source[key], keyPath + key, pathArray);
+            } else {
+                pathArray.push(keyPath + key);
+            }
+        });
+        return pathArray;
+    }
+    const excludeLanguages = ['en', 'es-ES'];
+    const languages = _.without(_.keys(originalTranslations.default), ...excludeLanguages);
+    const parentLanguage = originalTranslations.default.en;
```

Comment:
Kind of an odd name, I suggest:
```suggestion
    const mainLanguage = originalTranslations.default.en;
```